### ISSUE TITLE
`ranger --cmd "flat 1"` causes exception

### ISSUE THREAD

**Issue by FichteFoll** [ranger/ranger#1137]
#### Runtime Environment

- Operating system and version: Arch Linux
- Terminal emulator and version: alacritty, custom branch
- Python version: 3.6.4
- Ranger version/commit: a30b5e5b
- Locale: en_US.UTF-8

#### Current Behavior

Throws exception.

#### Expected Behavior

Runs specified command.

#### Context

Well, I want to open ranger in a directory and flatten it (by 1) immediately.

#### Steps to reproduce
1. `ranger --cmd "flat 1"`

#### Traceback

```
16:15:22 ERRO Notification: 'NoneType' object has no attribute 'unload'
16:15:22 ERRO 'NoneType' object has no attribute 'unload'
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/ranger/core/actions.py", line 251, in execute_console
    cmd.execute()
  File "/usr/lib/python3.6/site-packages/ranger/config/commands.py", line 1574, in execute
    self.fm.thisdir.unload()
AttributeError: 'NoneType' object has no attribute 'unload'
```

**Comment by FichteFoll** [comment]
Fixed via #1137.

**Comment by toonn** [comment]
Fixed in #1139 actually : p

**Comment by FichteFoll** [comment]
Ah yes, of course. #1137 is this issue.
